M2/practica/clientes.php

```diff
@@ -4,4 +4,24 @@
 
 $clientes = json_decode($json, true);
 
-echo json_encode($clientes);
\ No newline at end of file
+$i = 0;
+$cond = $_REQUEST['cond'];
+$in = $_REQUEST['in'];
+
+foreach ($clientes as $key => $value) {
+    if ($in <= $key) {
```

Comment:
Fijate que al usar foreach estas iterando por CADA elemento de clientes.json, que en esta practica no importa mucho porque sabemos que son solo 35, pero en una base de datos grande vos estarias salteando todos los elementos entre 0 y y $in y recien ahi empezas a agregar los elementos a la respuesta y al llegar a 10, cortas el foreach y devolves.
Lo que te sugiero, es que ya que sabes cual es el cliente a partir del cual empezar a buscar 5, que directamente empieces por ahi, con un for en vez de un foreach. Pero tene cuidado con que te pase un valor invalido en el ajax eh!